hawkular-alerts-actions/hawkular-alerts-actions-impl/src/main/java/org/hawkular/alerts/actions/standalone/ServiceNames.java

```diff
@@ -33,20 +35,24 @@
 
     private static String JNDI_ACTIONS_METRICS = "java:global/hawkular-metrics/hawkular-alerts/CassActionsServiceImpl";
     private static String JNDI_ACTIONS_STANDALONE = "java:global/hawkular-alerts/CassActionsServiceImpl";
-    private static String JNDI_DEFINITIONS_METRICS =
-            "java:global/hawkular-metrics/hawkular-alerts/CassDefinitionsServiceImpl";
+    private static String JNDI_DEFINITIONS_METRICS = "java:global/hawkular-metrics/hawkular-alerts/CassDefinitionsServiceImpl";
     private static String JNDI_DEFINITIONS_STANDALONE = "java:global/hawkular-alerts/CassDefinitionsServiceImpl";
 
     public enum Service {
-        ACTIONS_SERVICE,
-        DEFINITIONS_SERVICE
+        ACTIONS_SERVICE, DEFINITIONS_SERVICE
     }
 
     private static Map<Service, String> services;
 
     static {
         services = new HashMap<>();
-        if (System.getProperty(HAWKULAR_ALERTS_ACTIONS_ENV, STANDALONE).equals(STANDALONE)) {
+        String env = null;
+        try {
+            env = (String) new InitialContext().lookup("java:comp/env/" + HAWKULAR_ALERTS_ACTIONS_ENV);
```

Comment:
The reason is simple, because it didn't work for me.  If you can show me that it works as it was I'm happy to change it back.  If we do this in some othr places we should ensure that is working as expected.  We can discuss off-pr if you like.